.github/ng-renovate/runner-config.js

```diff
@@ -4,10 +4,7 @@ module.exports = {
   platform: 'github',
   forkMode: true,
   onboarding: false,
-  allowedPostUpgradeCommands: [
-    // See https://github.com/angular/angular/pull/47040.
-    '^yarn --cwd=aio/tools/examples/shared run sync-deps$',
-  ],
+  allowedPostUpgradeCommands: ['.'],
```

Comment:
Do we want to use `.*` or something? I don't know if Renovate expects a full match or if a partial is fine.

Reply:
No, I don't think Renovate expects a full match. For example, we were using ['^yarn run '][1] before and it worked fine afaict.

[1]: https://github.com/angular/dev-infra/blob/d2f9d2f493ec99f89a1db2777e2b27c1f5b278e9/.github/ng-renovate/runner-config.js#L7

Reply:
makes sense!